portal/common/static/404-page.template.html

```diff
@@ -10,6 +10,23 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>404 Walrus Site Not Found</title>
     </head>
+    <script>
+    function getPortalName() {
+        const url = window.location.href;
+        if (url.includes("walrus.site")) {
+            console.log("URL contains 'walrus.site'");
+            return ["https://walrus.site/", "Open walrus.site"];
+        }
+        return ["https://blob.store/", "Open blob.store"];
+    };
```

Comment:
Question: This will only work with `walrus.site` or `blob.store`. What if we wanted to support other (partner) portals? Can we use something like `getPortalUrl` in `common/lib/redirects.ts` (with the appropriate modifications ofc)

Reply:
Ah, now I get it, that makes sense. Alright I will make it this way 💯

Reply:
Looking more into it, I can't transfer the `getPortalUrl` from `redirects.ts` because it uses the `getDomain` function from `domain_parsing.ts`, which consequently uses the [parse-domain lib](https://www.npmjs.com/package/parse-domain).

To extract properly the portal domain, would require using our `getPortalUrl` because it splits reliably the suins/blobIDs from the portal.

This cannot be done with a few changes. On the other hand there is always the option to go with the old approach of exctracting the portal domain as the last 2 elements of the array that results with `<hostname>.split(".")`

Reply:
>On the other hand there is always the option to go with the old approach of extracting the portal domain as the last 2 elements of the array that results with <hostname>.split(".")

I'd go with this?

Reply:
Done ✔️